tests/test_enterprise/api/test_views.py

```diff
@@ -7400,6 +7401,100 @@ def test_list_learner_statuses(self):
         statuses = [result.get('member_status') for result in response.json().get('results')]
         assert statuses.sort() == ['accepted', 'pending'].sort()
 
+    def test_list_learners_filtered(self):
```

Comment:
nit: not blocking but might want to include a test with the page being passed in?

Reply:
I believe @katrinan029 already has a pagination test! https://github.com/openedx/edx-enterprise/blob/master/tests/test_enterprise/api/test_views.py#L7437-L7440